test/testcafe/spec/ChallengeCustomAppPush_spec.js

```diff
@@ -176,6 +178,7 @@ test
     await t.expect(answerRequestUrl2).eql('http://localhost:3000/idp/idx/challenge/poll');
   });
 
+/*
```

Comment:
will uncomment tests/some test assertions once we official support autopush with Custom push authenticator

Reply:
if that is the case, please add a jira number or a TODO so we know why this is commented out